static/src/js/components/ProjectPanels/AccessMethod.js

```diff
@@ -14,102 +14,155 @@ import './AccessMethod.scss'
  * @param {number} props.index - The index of the current collection.
  * @param {function} props.onSetActivePanel - Switches the currently active panel.
  */
-export const AccessMethod = ({ index, metadata, onSetActivePanel }) => {
-  const { id } = metadata
-  return (
-    <div className="access-method">
-      <ProjectPanelSection heading="Select Data Access Method">
-        <div className="access-method__radio-list">
-          <RadioList defaultValue="Download">
-            <Radio
-              id={`${id}_access-method__direct-download`}
-              name={`${id}_access-method__direct-download`}
-              value="Download"
-              checked
+export const AccessMethod = ({
+  index,
+  metadata,
+  onSelectAccessMethod,
+  onSetActivePanel
+}) => {
+  const {
+    id,
+    tags
+  } = metadata
+
+  const handleAccessMethodSelection = (method) => {
+    console.log('handleAccessMethodSelection', method)
+    onSelectAccessMethod({
+      collectionId: id,
+      selectedAccessMethod: method
+    })
+  }
+
+  const collectionCapibilities = tags['edsc.extra.serverless.collection_capabilities']
```

Comment:
I think you'll want to use the `getValueForTag` method mentioned in this review -- its within `util/tags.js`